src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -23,6 +26,24 @@ export interface StatementGroupModel {
     statementComposition: StatementGroupComposition;
 }
 
+export interface PermanentStatementGroup extends StatementGroupModelBase {
+    // Discriminator
+    type: StatementGroupType.permanent;
+
+    isActive?: boolean;
+    status: PermanentStatementGroupStatusType;
+}
+
+export interface CampaignStatementGroup extends StatementGroupModelBase {
+    // Discriminator
+    type: StatementGroupType.campaign;
+
+    campaignStart?: string;
+    campaignEnd?: string;
```

Comment:
I think having jsdocs at least for those would make sense, like to know what date format the backend uses